### ISSUE TITLE
ChatGPT-MD Not Able to Log In to ChatGPT

### ISSUE THREAD

**Issue by mark0mark0mark** [bramses/chatgpt-md#103]
I have the ChatGPT-MD installed in my Obsidian. I used it a few months ago many times without problems. My use case is pressing
ctrl+alt c with my cursor on a note page. That always worked before without error.

Now, I get an error popup that says: [ChatGPT MD] Error – You have exceeded your current quota, please check your plan and billing details.

I have a monthly subscription to ChatGPT Plus. I am currently using ChatGPT through their own website and have not seen any limits on my responses.

Things I have tried
I went to the OpenAI website and generated a new API key. I put that into ChatGPT-MD in Obsidian. That did not fix the problem.

I checked everywhere I could on the OpenAI website and all API usage info I could find say I have zero usage.

Is anyone else using this plugin? I see that it hasn’t been updated in a year. Does it still work?

Thanks,

Mark

**Comment by yossev** [comment]
Dev didnt take any pull reqs since april 2023 ( oldest pull req ).

**Comment by lukemt** [comment]
True, but the issue is probably related to the change in billing by openai.. Opaenai had me change to prepaid dev usage. 

First of all: ChatGPT Plus and the ChatGPT API are billed separately. This means the plus subscription does not pay for API usage (wich the plugin is using). 

When you sign up for an openai account you get some API credits for 3 months. After that you need to buy them.

**Comment by mark0mark0mark** [comment]
OK, that makes sense.  Thanks for the info!

**Comment by mark0mark0mark** [comment]
Thanks. I did that back when I replied in July. It was working then.
